cmd/docs/man_docs.go

```diff
@@ -123,25 +123,38 @@ func fillHeader(header *GenManHeader, name string) error {
 	return nil
 }
 
-func manPreamble(buf *bytes.Buffer, header *GenManHeader, cmd *cobra.Command, dashedName string) {
+func manPreamble(buf io.StringWriter, header *GenManHeader, cmd *cobra.Command, dashedName string) {
 	description := cmd.Long
 	if len(description) == 0 {
 		description = cmd.Short
 	}
 
-	buf.WriteString(fmt.Sprintf(`%% %s(%s)
-%% %s
-%% %s
-# NAME
-`, header.Title, header.Section, header.Source, header.Manual))
-	buf.WriteString(fmt.Sprintf("%s \\- %s\n\n", dashedName, cmd.Short))
-	buf.WriteString("# SYNOPSIS\n")
-	buf.WriteString(fmt.Sprintf("**%s**\n\n", cmd.UseLine()))
-	buf.WriteString("# DESCRIPTION\n")
-	buf.WriteString(description + "\n\n")
+	if _, err := buf.WriteString(fmt.Sprintf(`%% %s(%s)
```

Comment:
does `make docs` work after this change?

Reply:
yes, it is running but gave me warnings about some files that do not exist, I don't think those warnings are relevant to this change.
I didn't run make docs before, but it is working now